src/order/initial_seg.lean

```diff
@@ -209,8 +216,11 @@ theorem init [is_trans β s] (f : r ≺i s) {a : α} {b : β} (h : s b (f a)) :
 f.down.1 $ trans h $ f.lt_top _
 
 /-- A principal segment is in particular an initial segment. -/
+def to_initial_seg [is_trans β s] (f : r ≺i s) : r ≼i s :=
+⟨f.to_rel_embedding, λ a b, f.init⟩
```

Comment:
What's the purpose of this? Is this a mathlib4 backport?

Reply:
At the time it was for the convenience of use. But I can remove it now.